plugins/woocommerce/includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -123,6 +123,7 @@ public function read( &$coupon ) {
 			array(
 				'code'                        => $post_object->post_title,
 				'description'                 => $post_object->post_excerpt,
+				'status'	                  => $post_object->post_status,
```

Comment:
Fixes an indentation issue

```suggestion
				'status'	                      => $post_object->post_status,
```